pipedream.go

```diff
@@ -0,0 +1,59 @@
+package pd
+
+import (
+	"encoding/json"
+	"os"
+)
+
+var (
+
+	// Steps provides access to previous step data
+	// Unmarshal should be preferred with an explicit type
+	Steps map[string]interface{}
+
+	// StepsEnv is the environment variable name for previous pipedream steps
+	StepsEnv = "PIPEDREAM_STEPS"
+	// ExportsEnv is the environment variable name for exporting data to a file
+	ExportsEnv = "PIPEDREAM_EXPORTS"
+)
+
+// MustUnmarshal unmarshals the previous step data, panicking on error
+func MustUnmarshal(in interface{}) {
+	if err := Unmarshal(in); err != nil {
+		panic(err)
+	}
+}
+
+// Unmarshal unmarshals the previous step data, returning any errors encountered
+func Unmarshal(in interface{}) error {
+	pdSteps := getFromFile(StepsEnv, []byte("null"))
+	if err := json.Unmarshal(pdSteps, in); err != nil {
+		return err
+	}
+	return nil
+}
+
+// Export exports data for subsequent steps to use
+func Export(name string, value interface{}) {
+	export, _ := json.Marshal(value)
```

Comment:
check `err` here?

Reply:
Sure, this is just how it was prior to my changes. 
Do you want it to panic like the rest?